x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/BooleanFieldScript.java

```diff
@@ -84,4 +86,26 @@ public void value(boolean v) {
             script.emit(v);
         }
     }
+
+    public static class EmitValues {
+        private final BooleanFieldScript script;
+
+        public EmitValues(BooleanFieldScript script) {
+            this.script = script;
+        }
+
+        public void emitFromPath(String path) {
+            for (Object v : script.extractFromSource(path)) {
+                if (v instanceof Boolean) {
+                    script.emit((Boolean) v);
+                } else if (v instanceof String) {
+                    try {
+                        script.emit(Booleans.parseBoolean((String) v));
+                    } catch (Exception e) {
```

Comment:
what exception may we get here? IllegalArgumentException because the value is neither "true" nor "false"?